pandas/tools/tests/test_tile.py

```diff
@@ -272,6 +272,13 @@ def test_series_retbins(self):
                                     np.array([0, 0, 1, 1], dtype=np.int8))
         tm.assert_numpy_array_equal(bins, np.array([0, 1.5, 3]))
 
+    def test_qcut_duplicates_drop(self):
+        # GH 7751
+        values = [0, 0, 0, 0, 1, 2, 3]
+        cats = qcut(values, 3, duplicates='drop')
+        ex_levels = ['[0, 1]', '(1, 3]']
+        self.assertTrue((cats.categories == ex_levels).all())
```

Comment:
Can you add a test for `duplicates='raise'` (the default) as well? (assert that it raises a ValueError)